website/docs/DriverBinaries.md

```diff
@@ -3,7 +3,79 @@ id: driverbinaries
+To run automation based on the WebDriver protocol you need to have browser drivers set up that translate the automation commands and are able to execute them in the browser.
+
+## Automated setup
+
+With WebdriverIO `v8.14` and above there is no need to manually download and setup any browser drivers anymore as this is handled by WebdriverIO. All you have to do is specify the browser you want to test and WebdriverIO will do the rest. For more information on this feature, see [here](capabilities#automate-different-browser-channels).
+
+### Customizing the level of automation
+
+WebdriverIO's have three levels of automation:
+
+**1. Download and install the browser using [@puppeteer/browsers](https://www.npmjs.com/package/@puppeteer/browsers).**
+
+WebdriverIO will only do this if it cannot autodetect an installation of the browser with the `browserName`/`browserVersion` combination specified in the [capabilities](configuration#capabilities-1) configuration. If `browserVersion` is omitted, an arbitrary version will be used instead.
+
+:::caution
+
+Automated browser setup does not support `edge`. Currently, only `chrome` and `firefox` are supported.
+
+:::
+
+If you have a browser installation on a location that cannot be auto-detected by WebdriverIO, you can specify the browser binary which will disable the automated download and installation.
+
+```ts
+{
+    capabilities: [
+        {
+            browserName: 'chrome', // or 'firefox',
+            'goog:chromeOptions': { // or 'moz:firefoxOptions'
+                binary: '/path/to/chrome'
+            },
+        }
+    ]
+}
+```
+
+**2. Download and install the driver using [chromedriver](https://www.npmjs.com/package/chromedriver), [edgedriver](https://www.npmjs.com/package/edgedriver) or [geckodriver](https://www.npmjs.com/package/geckodriver).**
+
+WebdriverIO will always do this, unless driver [binary](capabilities#binary) is specified in the configuration:
+
+```ts
+{
+    capabilities: [
+        {
+            browserName: 'chrome', // or 'firefox', 'msedge', 'safari'
+            'wdio:chromedriverOptions': { // or 'wdio:geckodriverOptions', 'wdio:edgedriverOptions'
+                binary: '/path/to/chromedriver' // or 'geckodriver', 'msedgedriver'
+            }
+        }
+    ]
+}
+```
+
+:::info
+
+WebdriverIO won't automatically download Safari driver as it is already installed on macOS.
+
+:::
+
+:::caution
+
+If you specify a browser `binary` but omit the `browserVersion` and its corresponding driver `binary`, WebdriverIO will auto-determine the driver version to download based on the auto-detected browser installation. In some environments, the auto-detected browser version may differ from the specified binary. Therefore, it's recommended to always specify the browser version when specifying a browser binary and omitting the driver binary.
```

Comment:
> In some environments, the auto-detected browser version may differ from the specified binary.

Can you elaborate?

Reply:
I see, to summarise this: if you define either a browser or a driver binary but not both, you will likely end up having no matching binaries and your test will fail. Do you think we can formulate it like this somehow? I had to read the sentence a couple of times without really understanding what it means 🙈

Reply:
OK, I've reworded the paragraph. Hopefully this does the trick.